src/components/user.js

```diff
@@ -117,20 +117,55 @@ class User extends Emitter {
     Get stored user state from remote server.
     @private
     */
-    _getState(chat, callback) {
+    _getState(callback) {
```

Comment:
First try to get state from pubnub. If that doesn't work, get state from the server. If both don't work, oh well, no state.